code-challenges.test.js

```diff
@@ -13,106 +13,39 @@
-//After my expectation im going to write the name on my function and the variable that i want to change at the end i write what i want to see as my outcome.
-
-describe("remainderDiv", () => {
-  it("mixed data array and returns an array of only the remainder of the numbers when divided by 3",() => {
-    let hodgepodge1 = [23, "Heyyyy!", 45, -10, 0, "Yo", false]
-    let hodgepodge2 = [5, "Hola", 43, -34, "greetings", true]
-    expect(remainderDiv(hodgepodge1)).toEqual([ 2, 0, -1, 0 ]),
-    expect(remainderDiv(hodgepodge2)).toEqual([ 2, 1, 1, ])
-    })
-//REEEEED :(
-
-
-//I write this twice because my test keeps telling me that i have an unexpected token.
-// describe("remainderDiv", ()=>{
-//   it("mixed data array and returns an array of only the remainder of the numbers when divided by 3", ()=>{
-//   let hodgepodge1 = [23, "Heyyyy!", 45, -10, 0, "Yo", false]
-//   let hodgepodge2 = [5, "Hola", 43, -34, "greetings", true]
-//     expect(remainderDiv(hodgepodge1).toEqual([ 2, 0, -1, 0 ]),
-//     expect(remainderDiv(hodgepodge2).toEqual([ 2, 1, -1 ])
-//  })
-// })
-// b) Create the function that makes the test pass.
-//I start declaring my variable remainderDiv, I want to make a forloop so I can go in the array and just get
-// what i want that are the numbers after that i want to set my numbers the remainder or my array to be divided by 3.
-const remainderDiv = (array) => {
-  let divRemain = []
-  for(let i=0; i < array.length; i++){
-   if(typeof array[i] === `number` && divRemain.push(array[i]%3))
-  }
-      return divRemain
-}
-
-// My test didnt pass i still dont know where im making the error im going to have to re-do it.
+
+const hodgepodge1 = [23, "Heyyyy!", 45, -10, 0, "Yo", false]
 // Expected output: [ 2, 0, -1, 0 ]
+const hodgepodge2 = [5, "Hola", 43, -34, "greetings", true]
 // Expected output: [ 2, 1, -1 ]
+
+
+// b) Create the function that makes the test pass.
+
+
 // --------------------3) Create a function that takes in an array of numbers and returns the sum of all the numbers cubed.
 
 // a) Create a test with an expect statement using the variables provided.
 
-  describe("cubedArr", () =>{
-   it("array of numbers and returns the sum of all the numbers cubed", ()=> {
-    let cubeAndSum1 = [2, 3, 4]
-    let cubeAndSum2 = [0, 5, 10]
-    expect(cubedArr(cubeAndSum1)).toEqual([99]),
-    expect(cubedArr(cubeAndSum2)).toEqual([1125])
- })
-})
-// // // b) Create the function that makes the test pass.
-// //Im going to start declaring my variable cubedArr, im going to try a forloop here as well.
-// //I want to go throw my array and add al my values after that i want to multiply my total and set a variable to be my answer.
-//
-const cubedArr =(array) =>{
-  let addCube =0
-  for(let i= 0; i < array.length; i++){
-    array[i] = array [i]**3 // I try *3*3 first just to try but it didnt work.
-    addCube+=array[i]
-  }
-  return addCuben
-}
-
-
```

Comment:
Your logic is so close. There are a few ways you could approach this. This simplest might be `return [addCuben]`